test/specs/keys/keys_hash_crud_test.js

```diff
@@ -0,0 +1,95 @@
+import { login_page } from '../../../lib/pom/Login_page';
+import { main_page } from '../../../lib/pom/Main_page';
+import { keys_page } from '../../../lib/pom/Keys_page.js';
+import { Dashboard_connection } from '../../../lib/utils/api_connections/Dashboard_connection';
+import { newAPIdefinitionWithDefaults } from '../../../lib/utils/API_object_designer';
+
+const authTokenApi = {
+  "name": "authtoken",
+  "use_keyless": false,
+  "use_standard_auth": true
+};
+
+const apiKeysDetails = {
+  apiName: 'keys_crud',
+  keyExpiryTime: "1 hour",
+  alias:'test',
+};
+
+const updatedKeyDetails={
+  keyExpiryTimeUpdateValue: "6 hours",
+  aliasUpdate:'updateAlias',
+  metadataKey:'test',
+  metadataValue:'123',
+};
+
+
+describe('Create/update/delete keys without policy', () => {
+  const dashboard_connection = new Dashboard_connection();
+  let envDetails;
+
+  before(() => {
+    envDetails = setUpEnv();
+    login_page.open();
+    login_page.login(envDetails.userEmail, envDetails.userPassword);
+  });
+
+  it('Prerequisits: creating API definition via dashboard API', () => {
+      const body = newAPIdefinitionWithDefaults(authTokenApi);
+      dashboard_connection.createAPI(body, envDetails.userSecret);
+  });
+
+  it('User should be able to create new Key', () => {
+    main_page.openKeys();
+    keys_page.ADD_KEY_BUTTON.click();
+    keys_page.CHOOSE_API_TOGGLE.click();
+    wdioExpect(keys_page.CHOOSE_API_TABLE).toHaveTextContaining(authTokenApi.name);
+    keys_page.CHOOSE_API_TABLE.clickCellWithText(authTokenApi.name);
+    keys_page.CONFIGURATIONS_TAB_BUTTON.click();
+    keys_page.ALIAS_INPUT_FIELD.click();
+    keys_page.ALIAS_INPUT_FIELD.setValue(apiKeysDetails.alias);
+    keys_page.KEY_EXPIRE_DROPDOWN.click();
+    keys_page.KEY_EXPIRE_DROPDOWN.selectOption(apiKeysDetails.keyExpiryTime);
+    keys_page.ENABLE_DETAILED_LOGGING_BUTTON.click();
+    keys_page.CREATE_KEY_BUTTON.click();
+    keys_page.OK_BUTTON.click();
```

Comment:
can we add an assertion that key was really created?
it is nice to have assertion at the end of the step